src/LineItem.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EdifactParser;
+
+use EdifactParser\Segments\SegmentInterface;
+
+class LineItem
```

Comment:
If there is no inheritance, can we make this class final?
```suggestion
final class LineItem
```
Why? I wrote a [blog post about this topic](https://chemaclass.com/blog/final-classes/) that you might be interested 🧠

Reply:
Great point